src/main/java/com/team2502/robot2020/command/ToggleClimberLockCommand.java

```diff
@@ -0,0 +1,26 @@
+package com.team2502.robot2020.command;
+
+import com.team2502.robot2020.subsystem.ClimberSubsystem;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+public class ToggleClimberLockCommand extends InstantCommand {
+    private final ClimberSubsystem climber;
+
+    public ToggleClimberLockCommand(ClimberSubsystem climberSubsystem){
+        climber = climberSubsystem;
+        addRequirements(climberSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        if(climber.isRetracted() && DriverStation.getInstance().getMatchTime() < 35){
```

Comment:
is this confirmed to work?

Reply:
no
idk
I think it might have been there all of duluth

Reply:
i will delete anyways